mmv1/templates/terraform/examples/bigquery_connection_kms.tf.erb

```diff
@@ -0,0 +1,61 @@
+resource "google_sql_database_instance" "instance" {
+    name             = "<%= ctx[:vars]['database_instance_name'] %>"
+    database_version = "POSTGRES_11"
+    region           = "us-central1"
+    settings {
+		tier = "db-f1-micro"
+	}
+
+    deletion_protection  = "<%= ctx[:vars]['deletion_protection'] %>"
+}
+
+resource "google_sql_database" "db" {
+    instance = google_sql_database_instance.instance.name
+    name     = "db"
+}
+
+resource "random_password" "pwd" {
```

Comment:
could you pass in a password (even if it's random) instead of using the random provider? I think this would make the test deterministic, similar to https://github.com/hashicorp/terraform-provider-google-beta/blob/a84b92da47122442524846aeab327f48bdd24743/google-beta/services/databasemigrationservice/resource_database_migration_service_connection_profile_generated_test.go#L33

Reply:
Done!